test/es6/computedPropertyToString.js

```diff
@@ -34,15 +35,27 @@ var tests = [
                [name9()]() {}
                }
            var quxObj = new qux();
-           assert.areEqual("6() {}",           quxObj[name0()].toString(), "name0() adds 1+2+3, expecting 6");
-           assert.areEqual("11b() {}",         quxObj[name1()].toString(), "name1() returns number 1 plus string 1 and string b should evaluate to 11b");
-           assert.areEqual({}+date+{}+"() {}", quxObj[name2()].toString(), "name2() returns object + date + object should be toStrings of all three");
-           assert.areEqual("NaN() {}",         quxObj[name3()].toString(), "name3() returns NaN, which should be the name");
-           assert.areEqual("undefined() {}",   quxObj[name4()].toString(), "name4() returns undefined for the name");
-           assert.areEqual("null() {}",        quxObj[name5()].toString(), "name5() returns null as the name");
-           assert.areEqual("true() {}",        quxObj[name6()].toString(), "name6() returns true as the name");
-           assert.areEqual("false() {}",       quxObj[name7()].toString(), "name7() returns false as the name");
-           assert.areEqual(date+"() {}",       quxObj[name9()].toString(), "name8() returns a date as the name");
+           assert.areEqual("6",           quxObj[name0()].name, "name0() adds 1+2+3, expecting 6");
+           assert.areEqual("11b",         quxObj[name1()].name, "name1() returns number 1 plus string 1 and string b should evaluate to 11b");
+           assert.areEqual({}+date+{}+"", quxObj[name2()].name, "name2() returns object + date + object should be toStrings of all three");
+           assert.areEqual("NaN",         quxObj[name3()].name, "name3() returns NaN, which should be the name");
+           assert.areEqual("undefined",   quxObj[name4()].name, "name4() returns undefined for the name");
+           assert.areEqual("null",        quxObj[name5()].name, "name5() returns null as the name");
+           assert.areEqual("true",        quxObj[name6()].name, "name6() returns true as the name");
+           assert.areEqual("false",       quxObj[name7()].name, "name7() returns false as the name");
+           //assert.areEqual(sym,         quxObj[name8()].name, "name8() returns false as the name"); // Symbol doesn't play nicely

```

Comment:
Does this throw because implicit symbol conversion to a string throws?

Reply:
Yeah, I believe that's what happened. The original test case just left 8 out entirely, which confused me for a moment which is why I added it here with a note.